src/routes/jira/workspaces/jira-workspaces-get.test.ts

```diff
@@ -0,0 +1,248 @@
+		});
+		app.use(getFrontendApp());
+
+		await supertest(app)
+			.get("/jira/workspaces/search?searchQuery=incorrectorgname")
+			.query({
+				jwt
+			})
+			.expect(res => {
+				expect(res.status).toBe(400);
+				expect(res.text).toContain(NO_MATCHING_WORKSPACES);
+			});
+	});
+
+	it("Should return a single org if only one match is found", async () => {
+		app = express();
+		app.use((req, _, next) => {
+			req.log = getLogger("test");
+			next();
+		});
+		app.use(getFrontendApp());
+
+		await RepoSyncState.create({
+			...repo,
+			subscriptionId: sub.id
+		});
+
+		const workspaces = {
+			success:true,
+			workspaces: [
+				{
+					id: sub.id.toString(),
+					name: "atlassian",
+					canCreateContainer: false
+				}
+			]
+		};
+
+		await supertest(app)
+			.get("/jira/workspaces/search?searchQuery=atlas")
+			.query({
+				jwt
+			})
+			.expect(res => {
+				expect(res.text).toContain(JSON.stringify(workspaces));
+				expect(res.status).toBe(200);
+			});
+	});
+
+	it("Should return multiple matches but only return each org once", async () => {
+		app = express();
+		app.use((req, _, next) => {
+			req.log = getLogger("test");
+			next();
+		});
+		app.use(getFrontendApp());
+
+		const sub2 = await Subscription.install({
+			host: jiraHost,
+			installationId: 2345,
+			hashedClientKey: "key-123",
+			gitHubAppId: undefined
+		});
+
+		const sub3 = await Subscription.install({
+			host: jiraHost,
+			installationId: 3456,
+			hashedClientKey: "key-123",
+			gitHubAppId: undefined
+		});
+
+		await RepoSyncState.create({
+			subscriptionId: sub.id,
+			repoId: 1,
+			repoName: "github-for-jira",
+			repoOwner: "atlas",
+			repoFullName: "atlas/github-for-jira",
+			repoUrl: "github.com/atlas/github-for-jira"
+		});
+
+		await RepoSyncState.create({
+			subscriptionId: sub.id,
+			repoId: 2,
+			repoName: "github-for-jira",
+			repoOwner: "notamatch",
+			repoFullName: "notamatch/github-for-jira",
+			repoUrl: "github.com/notamatch/github-for-jira"
+		});
+
+		await RepoSyncState.create({
+			subscriptionId: sub2.id,
+			repoId: 3,
+			repoName: "github-for-jira",
+			repoOwner: "atlassian",
+			repoFullName: "atlassian/github-for-jira",
+			repoUrl: "github.com/atlassian/github-for-jira"
+		});
+
+		await RepoSyncState.create({
+			subscriptionId: sub2.id,
+			repoId: 3,
+			repoName: "github-for-jira",
+			repoOwner: "anotheratlasmatch",
+			repoFullName: "anotheratlasmatch/github-for-jira",
+			repoUrl: "github.com/atlassian/github-for-jira"
+		});
+
+		await RepoSyncState.create({
+			subscriptionId: sub3.id,
+			repoId: 3,
+			repoName: "github-for-jira",
+			repoOwner: "iworkatatlassian",
+			repoFullName: "iworkatatlassian/github-for-jira",
+			repoUrl: "github.com/iworkatatlassian/github-for-jira"
+		});
+
+		const workspaces = {
+			success:true,
+			workspaces: [
+				{
+					id: sub.id.toString(),
+					name: "atlas",
+					canCreateContainer: false
+				},
+				{
+					id: sub2.id.toString(),
+					name: "atlassian",
```

Comment:
Is this correct? The search parameter `atlas` should return both `atlassian` and `anotheratlasmatch`, isn't it? Why just `atlassian`?